node/network/bitfield-distribution/src/metrics.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
```

Comment:
```suggestion
// Copyright 2022 Parity Technologies (UK) Ltd.
```

Reply:
It's copied, so the license is correct as is